byteps/tensorflow/__init__.py

```diff
@@ -267,17 +273,77 @@ def get_slot_names(self, *args, **kwargs):
         def variables(self, *args, **kwargs):
             """Calls this same method on the underlying optimizer."""
             return self._optimizer.variables(*args, **kwargs)
+#x2682marklinexxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
#x2682marklinexxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

What is this?

Reply:
this was to make my diff tool behave. will remove.